chainer/training/extensions/nan_killer.py

```diff
@@ -0,0 +1,21 @@
+from chainer.training import extension
+
+
+class NaNKiller(extension.Extension):
+    """Trainer extension to raise RuntimeError if parameters contain NaN.
+
+    Although parameters including NaN are unnecessary for many developers,
```

Comment:
I think the word `developers` is ambiguous. How about `in most cases` instead of `for many developers`?